### PR TITLE
Add HW06.Separator,HW06.TheTriangleBuilder,HW06.ThirdArrayCreator

### PR DESCRIPTION
**Pull request by Vl92**

### CODE REVIEW

File: Lesson06/HW06.Separator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            string[] splitedArray = userImput.Split(';');
+            string poem = "";
+            for (int i = 0; i < splitedArray.Length; i++)
+            {
+                poem += (splitedArray[i] + "\n").Trim(' ').Replace('о', 'а');
+            }
+            Console.WriteLine(poem);
+
+            Console.ReadKey();
+        }
```
Comment (Onix-one) [inline Lesson06/HW06.Separator/Program.cs]: I don't see the third point of the task

File: Lesson06/HW06.TheTriangleBuilder/HW06.TheTriangleBuilder/Program.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            Console.WriteLine("Enter one or more lines of text (press CTRL+Z to exit):");
+            Console.WriteLine();
+            do
+            {
+                Console.Write("   ");
+                line = Console.ReadLine();
+                if (line != null)
+                    Console.WriteLine("      " + line);
+            } while (line != null);
+        }
```
Comment (Onix-one) [inline Lesson06/HW06.TheTriangleBuilder/HW06.TheTriangleBuilder/Program.cs]: What's that? Its not a solve of task...

File: Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Program.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            Console.WriteLine("enter a number:");
+            int n = Convert.ToInt32(Console.ReadLine());
+            for (int i = 1; i <= n; i++)
+            {
+                for (int x = i; x <= n; x++)
+                {
+                    Console.Write(" ");
+                }
+                for (int j = 1; j <= i; j++)
+                {
```
Comment (Onix-one) [inline Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Program.cs]: OMG. The solution is not from this task, and even if, it is still incorrect.

File: Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyCopyright("Copyright ©  2022")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Установка значения False для параметра ComVisible делает типы в этой сборке невидимыми
+// для компонентов COM. Если необходимо обратиться к типу в этой сборке через
+// из модели COM задайте для атрибута ComVisible этого типа значение true.
+[assembly: ComVisible(false)]
+
+// Следующий GUID представляет идентификатор typelib, если этот проект доступен из модели COM
```
Comment (Onix-one) [inline Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Properties/AssemblyInfo.cs]: Lesson 06 includes four tasks. I see only three

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson06/HW06.Separator/Program.cs (hunk 1 above)
Comment (Vl92) [reply Lesson06/HW06.Separator/Program.cs]: Done

File: Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Program.cs (hunk 3 above)
Comment (Vl92) [reply Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Program.cs]: done

File: Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (Vl92) [reply Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Properties/AssemblyInfo.cs]: Done

**Pull request merged**